### PR TITLE
Add blog post about shiny phyloseq IT

### PR DESCRIPTION
**Pull request by paulzierep**
@bgruening do we have a link to the Bioconductor hackathon? Who should be credited?

### CODE REVIEW

**Comment by paulzierep** [comment]
and maaslin2 apparently https://www.frontiersin.org/journals/cellular-and-infection-microbiology/articles/10.3389/fcimb.2022.854164/full -> need to get that

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: 'Shiny phyloseq interactive tool available on usegalaxy.eu - practical outcome of the Bioconductor Hackathon'
+date: '2024-26-07'
+tease: "We are thrilled to announce, that thanks to the help of the Bioconductor folks, we could deploy the Shiny phyloseq App as an interactive tool (IT) on usegalaxy.eu"
```
Comment (bernt-matthias) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:4]: folks -> community

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+hide_tease: false
+authors: Paul Zierep 
+subsites: [all-eu, global]
+---
+
+# Shiny phyloseq Interactive tool (IT)
+
+[Shiny apps](https://shiny.posit.co/) are web apps using R functionality, that give easy responsive access to R packages. 
+Interactive tools are a great way to work with data interactively and responsive using Galaxy. In theory all shiny apps could become ITs, but so far the
```
Comment (bernt-matthias) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:13]: Introduce abbreviation IT

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+authors: Paul Zierep 
+subsites: [all-eu, global]
+---
+
+# Shiny phyloseq Interactive tool (IT)
+
+[Shiny apps](https://shiny.posit.co/) are web apps using R functionality, that give easy responsive access to R packages. 
+Interactive tools are a great way to work with data interactively and responsive using Galaxy. In theory all shiny apps could become ITs, but so far the 
+wrapping and deployment of shiny apps as ITs was technically challenging. Thanks to a Hackathon together with folks from the Bioconductor 
+communities best-practices for the development of ITs using Shiny apps were developed, focusing mainly on the development of a docker image, that can be used
```
Comment (bernt-matthias) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:15]: I would be interested to know more about how you construct the docker image. Is there a link that you could add here / share with me?

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+hide_tease: false
+authors: Paul Zierep 
+subsites: [all-eu, global]
+---
+
+# Shiny phyloseq Interactive tool (IT)
+
+[Shiny apps](https://shiny.posit.co/) are web apps using R functionality, that give easy responsive access to R packages. 
+Interactive tools are a great way to work with data interactively and responsive using Galaxy. In theory all shiny apps could become ITs, but so far the 
+wrapping and deployment of shiny apps as ITs was technically challenging. Thanks to a Hackathon together with folks from the Bioconductor
```
Comment (bernt-matthias) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:14]: folks -> community

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+* scatter plots
+* bar charts
+
+<img src="./Barplot.png" style="max-width: 400px" alt="Barplot" />
+
+The tools is available on usegalaxy.eu (https://usegalaxy.eu/?tool_id=interactive_tool_phyloseq&version=latest) and was integrated into a [dada2 based GTN tutorial](https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/dada-16S/tutorial.html) by Bérénice Batut.
+
+## Data upload from Galaxy
+
+Although the IT worked with the original shiny app. The shiny-phyloseq app did not allow to upload amplicon data using command line arguments, which makes it difficult to allow for data input from Galaxy directly (downloading data from the Galaxy Histroy and uploading to the IT via the web frontend would work though). Therefore, the tool was forked and modified to allow for additional data input. This functional adaptation of shiny-apps to allow for non-web based data upload is the only requirement to make any shiny app compatible with Galaxy based data upload.
```
Comment (bernt-matthias) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:47]: Histroy -> History

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+authors: Paul Zierep 
+subsites: [all-eu, global]
+---
+
+# Shiny phyloseq interactive tool (IT)
+
+[Shiny apps](https://shiny.posit.co/) are web apps using R functionality, that give easy responsive access to R packages. 
+Interactive tools are a great way to work with data interactively and responsive using Galaxy. In theory all shiny apps could become ITs, but so far the 
+wrapping and deployment of shiny apps as ITs was technically challenging. Thanks to a Hackathon together with members from the Bioconductor 
+communities best-practices for the development of ITs using Shiny apps were developed, focusing mainly on the development of a docker image, that can be used
```
Comment (bgruening) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:15]:
```suggestion
communities best-practices for the development of ITs using Shiny apps were developed, focusing mainly on the development of a Docker image, that can be used
```

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+as a starting point to add any any shiny app. 
+As a proof-of-concept the [shiny-phyloseq app](https://github.com/joey711/shiny-phyloseq) has been wrapped as IT. 
+A fork of this docker image tailored for the shiny app is available here: https://github.com/paulzierep/docker-phyloseq.
```
Comment (bgruening) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:18]:
```suggestion
A fork of this Docker image tailored for the shiny app is available here: https://github.com/paulzierep/docker-phyloseq.
```

File: content/news/2024-07-26-shiny-phyloseq-on-EU/index.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+The image can be run locally to test the app and then must be deployed to quay.io.
```
Comment (bgruening) [inline content/news/2024-07-26-shiny-phyloseq-on-EU/index.md:19]:
```suggestion
The image can be run locally to test the app and then must be deployed to [quay.io](https://quay.io) or any other Docker registry. 
```

**Comment by bgruening** [comment]
No blog yet for the hackathon.

Does the gif needs an update?

**Comment by bgruening** [comment]
Charlotte Soneson, Hans-Rudolf Hotz, Federico Marini

**Comment by paulzierep** [comment]
good to go @bgruening

**Pull request merged**